internal/commands/server/k8s_e2e_test.go

```diff
@@ -1694,13 +1974,20 @@ func checkTCPRoute(t *testing.T, port int, expected string, message string) {
 			Port: port,
 		})
 		if err != nil {
+			log.Print("TCP_DIAL_ERROR: ", err)
```

Comment:
Were these `Prints` intended to stick around?

Reply:
I found them quite useful when writing the tests (and wasted time fiddling around with like three different print functions to find something that would actually show up in test output), had an open question in PR description about keeping these or changing to `t.Log`

Reply:
Whoops, I missed that one - my bad. I'm fine leaving them if they're helpful but would switch to `t.Log[f]`